test/parallel_api/iterator/iterators.pass.cpp

```diff
@@ -234,6 +234,13 @@ void test_transform_effect(VecIt1 first1, VecIt1 last1, VecIt2 first2) {
         }
 }
 
+struct ref_transform_functor {
```

Comment:
We need this functor to run fill algorithm with transform iterators. Operator() should return lvalue reference.

Reply:
Please, add this comment to the code